src/util/heap/freelistpageresource.rs

```diff
@@ -114,12 +114,11 @@ impl<VM: VMBinding> PageResource<VM> for FreeListPageResource<VM> {
             .space
             .unwrap()
             .grow_space(rtn, bytes, new_chunk);
-        self.common()
-            .space
-            .unwrap()
-            .common()
-            .mmapper
-            .ensure_mapped(rtn, required_pages);
+        self.common().space.unwrap().common().mmapper.ensure_mapped(
+            rtn,
+            required_pages,
+            &|_| unreachable!(),
```

Comment:
Why is the callback not invoked for `FreelistPageResource`?

Reply:
This is just my temporary metadata quick fix. I only made it work with 64bit contiguous space, so this code will never be executed when running gencopy with openjdk. Anyway, we need to fix this after figuring out a better approach for the fix.

Reply:
Alright. You requested reviews so I assumed it was ready for reviewing.

Reply:
Fixed. Now support FreelistPageResource as well.